src/main/java/com/faforever/client/chat/ChatUserContextMenuController.java

```diff
@@ -135,4 +180,21 @@ void onKick() {
   void onBan() {
     // FIXME implement
   }
+
+  @FXML
+  void onJoinGame() {
+    GameInfoBean gameInfoBean = gameService.getByUid(playerInfoBean.getGameUID());
+    gameService.joinGame(gameInfoBean,null,new Callback<Void>() {
```

Comment:
Please enable the following settings in your commit dialog:
* Reformat code
* Rearrange code
* Optimize imports